src/components/header/privateHeader.js

```diff
@@ -7,6 +7,8 @@ import { orderControl, adminControl } from "../protectedRoute/permissions";
 function Privateheader(params) {
   function logOut() {
     AuthService.logout();
+    console.log("passed");
```

Comment:
I know this has already been merged in but could you remove this console.log statement when you have a chance :)